htdocs/admin/confexped.php

```diff
@@ -70,7 +70,7 @@
 	header("Location: confexped.php");
 	exit;
 } elseif ($action == 'disable_delivery') {
-	dolibarr_del_const($db, "MAIN_SUBMODULE_DELIVERY", $conf->entity);
+	dolibarr_set_const($db, "MAIN_SUBMODULE_DELIVERY", 0, 'chaine', 0, '', $conf->entity);
```

Comment:
I see no reason to not remive this const completely

Reply:
You are right, I reverted this commit.

Reply:
You are right, I reverted the related commit.